src/binding/petsc4py/src/petsc4py/PETSc/Vec.pyx

```diff
@@ -152,11 +154,11 @@ cdef class Vec(Object):
         Parameters
         ----------
         viewer
-            The viewer instance, defaults to printing vector contents.
+          A `Viewer` instance or `None` for the default viewer.
```

Comment:
I know Sphinx renders it well, but why not keep it consistent with how it is done elsewhere?

```suggestion
            A `Viewer` instance or `None` for the default viewer.
```

Reply:
How about using a code formatter?

Reply:
docformatter seems quite good

```
(petsc4py-docs) [szampini@localhost petsc4py]$ docformatter --blank src/petsc4py/PETSc/Vec.pyx 
--- before/src/petsc4py/PETSc/Vec.pyx
+++ after/src/petsc4py/PETSc/Vec.pyx
@@ -1485,7 +1485,8 @@
         handle: Any,  # FIXME What type hint is appropriate?
         mode: AccessModeSpec = 'rw',
     ) -> None:
-        """Restore a pointer to the device buffer obtained with `getCUDAHandle`.
+        """Restore a pointer to the device buffer obtained with
+        `getCUDAHandle`.
 
         Not collective.
 
@@ -1693,7 +1694,8 @@
         return memhdl
 
     def restoreCLMemHandle(self) -> None:
-        """Restore a pointer to the OpenCL buffer obtained with `getCLMemHandle`.
+        """Restore a pointer to the OpenCL buffer obtained with
+        `getCLMemHandle`.
 
         Not collective.
 
@@ -1911,7 +1913,8 @@
         CHKERR( VecTDotBegin(self.vec, vec.vec, &sval) )
 
     def tDotEnd(self, Vec vec) -> Scalar:
-        """Finish computing the indefinite dot product initiated with `tDotBegin`.
+        """Finish computing the indefinite dot product initiated with
+        `tDotBegin`.
 
         Collective.
 
@@ -3147,6 +3150,7 @@
         --------
         createGhost, ghostUpdateBegin, ghostUpdateEnd
         petsc.VecGhostGetLocalForm, petsc.VecGhostRestoreLocalForm
+
         """
         return _Vec_LocalForm(self)
 
@@ -3174,7 +3178,8 @@
         addv: InsertModeSpec = None,
         mode: ScatterModeSpec = None,
     ) -> None:
-        """Finish updating ghosted vector entries initiated with `ghostUpdateBegin`.
+        """Finish updating ghosted vector entries initiated with
+        `ghostUpdateBegin`.
 
         Neighbour-wise collective.
 

```

Reply:
I mean, we can manually edit the first line to fit in the character limit and use it